src/cqww_simulator.c

```diff
@@ -149,10 +149,19 @@ void cqww_simulator(void) {
 	strcpy(callcpy, CALLMASTERARRAY(callnumber));
 	getctydata(callcpy);
 
-	char *str = g_strdup_printf("%s%s%s",
-				    "---" + 3 - slow,
```

Comment:
I think writing `&"---"[3-slow]` would fix the warning (can't test it with gcc10 at the moment)

Reply:
I will redo this as Zoli has suggested.